.travis.yml

```diff
@@ -39,10 +39,8 @@ matrix:
       compiler: gcc
 
 install:
-  - if [ "$TRAVIS_OS_NAME" = osx ]; then
-      HOMEBREW_NO_AUTO_UPDATE=1
-      brew install gettext openssl;
-    fi
+  - if [ "$TRAVIS_OS_NAME" = osx ]; then brew update && brew bundle; fi
```

Comment:
I'm not familiar with brew, does it upgrade some part of itself when doing the update?

Reply:
sorry for the late reply, got caught up.

`brew` on `travis` currently has a bug, you need to update it before it can be useful